documentation/library_guide/dynamic_selection_api/fixed_resource_policy.rst

```diff
@@ -0,0 +1,236 @@
+fixed_resource_policy
+#####################
+
+Always returns the same resource selection. ``fixed_resource_policy`` 
+is designed for two primary scenarios: (1) debugging the use of dynamic selection 
+and (2) special casing a dynamic selection capable application for a specific 
+resource is always best on a specific platform.
+
+.. code:: cpp
+
+  namespace oneapi::dpl::experimental {
+  
+    template<typename Backend=sycl_backend> 
+    class fixed_resource_policy {
+    public:
+      // useful types
+      using resource_type = typename Backend::resource_type;
+      using wait_type = typename Backend::wait_type;
+      
+      class selection_type {
+      public:
+        fixed_resource_policy<Backend> get_policy() const;
+        resource_type unwrap() const;
+      };
+      
+      // constructors
+      fixed_resource_policy(deferred_initialization_t);
+      fixed_resource_policy(::std::size_t offset=0);

```

Comment:
I would suggest to put spaces everywhere in similar places for readability (yes, the matter of taste but I believe it's better).

Also I would suggest removing leading `::` before `std`. It's our implementation detail and doesn't bring any value to a user.

Something like:
```suggestion
      fixed_resource_policy(std::size_t offset = 0);

```

Reply:
Made consistently std::size_t across all files.